time stamp
uint32 left_laps_p
uint32 right_laps_p
uint32 left_laps_n
uint32 right_laps_n